Project2015To2017/Transforms/Vs15TransformationSet.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Project2015To2017.Transforms
+{
+	public class Vs15TransformationSet : ITransformationSet
+	{
+		public static readonly Vs15TransformationSet TrueInstance = new Vs15TransformationSet();
+
+		public static readonly ITransformationSet Instance = new ChainTransformationSet(
+			BasicReadTransformationSet.Instance,
+			TrueInstance);
+
+		private Vs15TransformationSet()
+		{
+		}
+
+		public IReadOnlyCollection<ITransformation> Transformations(
+			ILogger logger,
+			ConversionOptions conversionOptions)
+		{
+			return new ITransformation[]
+			{
+				// Generic
+				new TargetFrameworkReplaceTransformation(
+					conversionOptions.TargetFrameworks,
+					conversionOptions.AppendTargetFrameworkToOutputPath),
+				new PropertyDeduplicationTransformation(),
+				new PropertySimplificationTransformation(),
+				new PrimaryProjectPropertiesUpdateTransformation(),
+				new EmptyGroupRemoveTransformation(),
+				// VS15 migration
+				new TestProjectPackageReferenceTransformation(logger),
+				new AssemblyFilterPackageReferencesTransformation(),
+				new AssemblyFilterHintedPackageReferencesTransformation(),
+				new AssemblyFilterDefaultTransformation(),
+				new ImportsTargetsFilterPackageReferencesTransformation(),
+				new FileTransformation(logger),
+				new XamlPagesTransformation(logger),
```

Comment:
Looks like this is missing `AssemblyAttributeTransformation` but maybe I have misunderstood how this is all working. I'm finding it much harder to follow what's going on

Reply:
I see now that the two are combined above in `Instance`. So then what does `TrueInstance` mean?